packages/ui/src/Select/helpers.ts

```diff
@@ -5,6 +5,7 @@ import { canUseDOM } from '../utils/ssr'
 export type SelectFieldOption<V> = {
   label: string
   value: V
+  sideNote?: string
```

Comment:
`trailingNote` sounds better and we have same naming in TextField - `trailingIcon`. WDYT?

Reply:
I had hard times finding a name for this :) Changed it to `trailingNote` 👍